libafl/src/executors/forkserver.rs

```diff
@@ -378,14 +378,24 @@ pub trait HasForkserver {
 pub struct TimeoutForkserverExecutor<E: Debug> {
     executor: E,
     timeout: TimeSpec,
+    signal: Signal,
 }
 
 impl<E: Debug> TimeoutForkserverExecutor<E> {
     /// Create a new [`TimeoutForkserverExecutor`]
-    pub fn new(executor: E, exec_tmout: Duration) -> Result<Self, Error> {
+    pub fn new(
+        executor: E,
+        exec_tmout: Duration,
+        kill_signal: Option<Signal>,
```

Comment:
Instead of changing the existing API, I would add an additional function like `with_signal`

Reply:
I think I know how this should be implemented, and it seems reasonable. Will push a new version tomorrow, and rebase if needed.

Reply:
And, last nit, now there is no real reason for the param to be `Option` ;)
Else, good first rusting

Reply:
And `new` could return `with_signal` if you want to be fancy

Reply:
The new commit includes some duplicated code, I'm working on that...